lib/rubygems/core_ext/kernel_warn.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+if RUBY_VERSION >= "2.5"
+  module Kernel
+    path = "#{__dir__}/"
+    original_warn = instance_method(:warn)
+    Module.new {define_method(:warn, original_warn)}
+    original_warn = method(:warn)
```

Comment:
This and line 6 appear to conflict. Which is it supposed to be?

Reply:
comment out 6 & 7 and see what happens...